packages/maskbook/src/plugins/Trader/trader/useTradeComputed.ts

```diff
@@ -91,7 +94,6 @@ export function useTradeComputed(
         outputToken,
     )
 
-    // dodo
```

Comment:
why remove the line?

Reply:
this comment doesnt make sense given line below, but agree it should not be part of my PR.
do you want me to revert it ?

Reply:
fixed